docs/creating-routes.md

```diff
@@ -0,0 +1,45 @@
+You may want to make prototypes that are more complex than simple HTML files. For example, you may want to respond to input from a form, and show different pages based on answers given by the user.
+
+To do this you will need to create 'routes' - rules for the server to respond to certain URLs.
+
+For example, with a route of `/sample` the URL is:
+
+    http://localhost:3000/sample
+    
+All routes for the application are kept in the [routes.js](../app/routes.js) file. They follow this format:
+
+    verb(route, callback(request, response) {
+        response.render(template, data);
+    });
+
+Let's break this down into bits:
+
+* **verb** : the type of request ('get' or 'post')
+* **route** : the route section of the URL as explained above
+* **callback** : a function that contains the code executed when that route is requested
+* **request** : the 1st parameter sent to the callback, an object representing the HTTP request made
+* **response** : the 2nd parameter sent to the callback, an object representing the HTTP response that will be sent
+* **response.render** : method of the response object used to create a page to send back to the browser that made the request
+* **template** : the 1st parameter sent to response.render, the name of the template file used to render the page, minus its `.html` extension
+* **data** : the 2nd parameter sent to response.render, an object containing variables to send into the template
+
+So as an example, a request for the URL `http://localhost:3000/sample` has this route:
+
+    get('/sample', function(req, res) {
+        res.render('sample', { 'assetPath' : '/public/' });
```

Comment:
assetPath is no longer sent manually

I think sample should live in `/views/examples`, and we could send an example variable to it, like an H1 or something